manifests/kustomize/base/tekton/pipeline-loops/200-serviceaccount.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The Knative Authors
```

Comment:
2020 -> 2021

Reply:
I removed all the license headers to be consistent with the rest of kfp manifests.